test/metric.jl

```diff
@@ -1,5 +1,6 @@
 using Test
 using ManifoldsBase
+import ManifoldsBase: inner, change_representer, change_metric
```

Comment:
This import doesn't seem to be used.

Reply:
ah yes, I first thought I write the tests there but when I then added the code directly to the DefaultManifold, I forgot to remove that line again.